src/mam4xx/conversions.hpp

```diff
@@ -56,7 +58,9 @@ KOKKOS_INLINE_FUNCTION Real number_conc_from_mmr(Real mmr, Real molecular_wt,
 /// @param [in] molecular_wt The molecular weight of the species/mixture
 /// [kg/kmol]
 KOKKOS_INLINE_FUNCTION Real mmr_from_vmr(Real vmr, Real molecular_wt) {
-  const auto mw_dry_air = Constants::molec_weight_dry_air;
+  // we convert here since haero::Constants uses SI units.
```

Comment:
I thought these functions would use densities, not molecular weights:

vmr [m3 aero / kg dry air] *  (density aerosol [kg aerosol / m3 aerosol]) = mmr [kg aerosol / kg dry air]

Reply:
Given the definitions above, that makes perfect sense to me--will change!